app.js

```diff
@@ -1,185 +1,249 @@
-var timeLabelsArray = ['10AM: ','11AM: ','12PM: ', '1PM: ', '2PM: ', '3PM: ', '4PM: ', '5PM: ', 'SUM: ']
-var pikePlace = {
-  min: 17,
-  max: 88,
-  avg: 5.2,
-  randomOutput: function(){
-    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
-  },
-  dailyRandomOutput: function(){
-    var cookiesArray = [];
-    for(i = 0; i < 8; i++){
-      var arrayValue = this.randomOutput();
-      cookiesArray.push(arrayValue);
-    }
-    return cookiesArray;
-  },
-  randomOutputSum: function(){
-    var arraySum = this.dailyRandomOutput();
-    var sum = 0;
-    for(i = 0; i < arraySum.length; i++){
-      sum += arraySum[i];
-    }
-    arraySum.push(sum);
-    console.log(arraySum);
-    return arraySum;
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('pike_place');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
-    };
-  }
-};
-pikePlace.renderManipulation();
+// var timeLabelsArray = ['10AM: ','11AM: ','12PM: ', '1PM: ', '2PM: ', '3PM: ', '4PM: ', '5PM: ', 'SUM: ']
 
-var seaTacAirport = {
-  min: 6,
-  max: 24,
-  avg: 1.2,
-  randomOutput: function(){
+function Store(name, min, max, avg){
+  this.name = name;
+  this.min = min;
+  this.max = max;
+  this.avg = avg;
+  this.randomOutput = function(){
     return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
-  },
-  dailyRandomOutput: function(){
+  };
+  this.dailyRandomOutput = function(){
     var cookiesArray = [];
     for(i = 0; i < 8; i++){
       var arrayValue = this.randomOutput();
       cookiesArray.push(arrayValue);
     }
     return cookiesArray;
-  },
-  randomOutputSum: function(){
-    var arraySum = this.dailyRandomOutput();
-    var sum = 0;
-    for(i = 0; i < arraySum.length; i++){
-      sum += arraySum[i];
-    }
-    arraySum.push(sum);
-    console.log(arraySum);
-    return arraySum;
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('seatac_airport');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
+  };
+  this.randomOutputSum = function(){
+      var arraySum = this.dailyRandomOutput();
+      var sum = 0;
+      for(i = 0; i < arraySum.length; i++){
+        sum += arraySum[i];
+      }
+      arraySum.push(sum);
+      console.log(arraySum);
```

Comment:
Would encourage you to get in the habit of pulling out console logs before submissions. It's fine for 201, but you typically never want console logs outside of the development environment. (I realize I'm being unnecessarily picky, but it's something worth a mention).